mmv1/templates/terraform/datasource_iam.html.markdown.erb

```diff
@@ -36,8 +36,8 @@
     directly inserted from YAML.
 -%>
 <%
-  tf_product = (object.__product.legacy_name || product_ns).underscore
-  product_ns_display = (object.__product.legacy_name || product_ns).camelize
+  tf_product = (object.__product.legacy_name || object.product_ns).underscore
```

Comment:
I wonder if we can directly use `object.__product.name` instead of the new variable `object.product_ns` here and in other places?

Reply:
This makes sense! The logic seemed more than just `object.__product.name` while refactoring, but looking at it now, that is all it is. 

Reworked.

Reply:
Thanks for working on it.